mod/carrierwave/lib/carrier_wave/card_mount.rb

```diff
@@ -29,13 +29,13 @@ def mount_uploader column, uploader=nil, options={}, &block
           remove_#{column}!
         end
 
-        event :mark_remove_#{column}_false_event, :finalize, on: :update do
-          mark_remove_#{column}_false
-        end
+        # event :mark_remove_#{column}_false_event, :finalize, on: :update do
+        #   mark_remove_#{column}_false
+        # end
```

Comment:
I don't know how important this is. This event was triggering the `#write_temporary_identifier` thing, which was overwriting the correct `#db_content_value` with the incorrect one. (You can reproduce by uncommenting and running `coded_spec.rb`)

I worry this omission could have unintended side effects.

Reply:
I think just removing breaking code is not a good approach. This section should be adjusted to the changes in the carrierwave code. What we do here is replacing the activerecord hooks in https://github.com/carrierwaveuploader/carrierwave/blob/d27c9871c83613717bd69d31da7d23450bf266c9/lib/carrierwave/orm/activerecord.rb#L29 with card events